_aggregations/metric/median-absolute-deviation.md

```diff
@@ -128,7 +103,10 @@ GET opensearch_dashboards_sample_data_flights/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+The estimation error depends on the dataset, but is usually below 5%, even for `compression` values as low as 100. (The low example value of `10` is used here to illustrate the tradeoff effect, and is not recommended.)
+
+Note the different value of the estimated parameter in the following response. For reference, the true median absolute deviation of `DistanceMiles` is `1831.076904296875`:
```

Comment:
`1831.076904296875` - get the correct number from the very first example response.

-----
Note the increased performance (`took` time)..... and the less accurate value of estimated parameter...

Reply:
```suggestion
Note the decreased computation time (`took` time) and the slightly less accurate value of the estimated parameter in the following response. For reference, the true median absolute deviation of `DistanceMiles` is `1831.076904296875`:
```

The first example actually is a slightly inaccurate estimate. I got (what I presume is the) true value by increasing `compression` to where the estimate stopped changing at large values.

Reply:
Yep, you are right here. Thanks for taking the efforts to run this exercise.

I'm just wondering should we still call it `true` MAD or something else - since this is like a more accurate estimation, but still an estimated number with a high accuracy parameter.